README.md

```diff
@@ -44,10 +44,10 @@ It's recommended to create a new virtual environment to install the SDK. You can
 
 ```bash
 # Create the virtual environment
-python3 -m venv venv
```

Comment:
This must be python3 as python 2 (accessed by `python` outside a venv) is no longer packaged with macOS

Reply:
when installing python3, `python` is automatically linked to `python3`. Generally, we should use the generic version.

`% which python
python: aliased to python3

% which python3
/opt/homebrew/bin/python3

% python --version
Python 3.13.1
`